source/mobile/mobile-appconfig.rst

```diff
@@ -50,3 +50,22 @@ As part of AppConfig, EMM administrators can set the following additional config
 4. **Remotely Wipe App:** Use the EMM tool to distribute the app to devices as a managed application so it can be remotely wiped. If the app was previously installed, mark it so the EMM converts the app to a managed app.
 
 Other configurations may be available depending on your EMM provider.
+
+Adding Mattermost mobile apps to your EMM provider
+------------------------------------------------------------------------
+
+Because the Mattermost mobile apps have support for AppConfig, you can deploy the apps found in Google Play and the Apple App Store, or you can choose to deploy them as in-house apps.
```

Comment:
I think this is already mentioned in the first section? `Mattermost mobile apps can be configured in your EMM solution using AppConfig with the apps on the public app stores (Google Play and Apple App Store), or as an "in-house app" you compile yourself.`

Maybe we can combine this section with it